components/common/chip/chip.tsx

```diff
@@ -48,20 +49,23 @@ const StyledChip = styled.button<{ otherOption?: boolean }>`
     &:active {
         background-color: rgba(var(--mention-highlight-link-rgb), 0.08);
     }
-
 `;
 
 export default class Chip extends React.PureComponent<Props> {
     onClick = (e: React.MouseEvent) => {
         e.preventDefault();
-        this.props.onClick();
+        e.stopPropagation();
+        if (this.props.onClick) {
+            this.props.onClick();
+        }
```

Comment:
since this is a `tsx` file we can shorten this:

```suggestion
        this.props.onClick?.();
```

Reply:
ok